docker/labels.go

```diff
@@ -11,10 +11,11 @@ type Label string
 
 // Libcompose default labels.
 const (
-	NAME    = Label("io.docker.compose.name")
-	PROJECT = Label("io.docker.compose.project")
-	SERVICE = Label("io.docker.compose.service")
-	HASH    = Label("io.docker.compose.config-hash")
+	NAME     = Label("io.docker.compose.name")
+	PROJECT  = Label("io.docker.compose.project")
+	SERVICE  = Label("io.docker.compose.service")
+	HASH     = Label("io.docker.compose.config-hash")
+	RECREATE = Label("io.docker.compose.recreate")
```

Comment:
@dnephin, Regarding the RECREATE label: true it's not supported by docker-compose now. But if docker-compose adds similar behaviour, it'll need a similar label. 

@ibuildthecloud and I need this feature for RancherOS, because we're using libcompose to manage system services, and some of them (like console) need to be persistent while the others are recreated on boot. Sure we can re-implement parts of libcompose (or the whole thing, as was the case before @ibuildthecloud factored it out from RancherOS and *created* libcompose), but that doesn't sound like progress.

I'd really like to know what's wrong with this feature (marking services with desired recreate behaviour) that makes it unacceptable here. If someone uses this "recreate" label in their compose template with libcompose, does it hurt docker-compose in any way? Sure they might ask docker-compose maintainers to add support for it in a future version, but is it a bad thing?

Another question: is there an official document for libcompose that states it should be a subset of docker-compose, feature-wise?